DataScience/Experimentation.py

```diff
@@ -200,14 +200,19 @@ def parse_cb_types(val):
         raise argparse.ArgumentTypeError('Input "{}" is an invalid cb_types input string - Valid cb_types are mtr, dr, ips'.format(val))
     return cb_types
 
-def generate_predictions_files(log_fp, policies):
+def create_prediction_command(policy_command, log_type):
+    if log_type == 'ccb':
+        return policy_command.replace('--ccb_explore_adf --epsilon 0', '--ccb_explore_adf --epsilon 0.2')
+    else:
+        return policy_command.replace('--cb_adf', '--cb_explore_adf --epsilon 0.2');
 
+def generate_predictions_files(log_fp, policies, log_type):
     predictions_files = []
     data = {}
     data['policies'] = []
-    Logger.info('Generating predictions files (using --cb_explore_adf) for {} policies:'.format(len(policies)))
+    Logger.info('Generating predictions files for {} '.format(log_type) + 'for {} policies:'.format(len(policies)));
```

Comment:
remove `;`

Reply:
I'll fix this in the next PR coming shortly.